KLR/Serde/Basic.lean

```diff
@@ -28,6 +28,11 @@ class FromCBOR (a : Type u) where
 
 open FromCBOR (parse)
 
+-- This is used by the derived instances
+def parseCBOR' [FromCBOR a] (arr : ByteArray) (size : Nat) : Err (ByteArray × Nat × a) :=
```

Comment:
I had this too. Annoying we can't make it private. Briefly wondered if we should come up with some convention like `def _parseCBOR_deriving_ ...` or something, but lean4 doesn't do it so probably fine.